server/data/badger-brain-test.js

```diff
@@ -0,0 +1,16 @@
+import {
+  getSiteState,
+  badgerBrainURL,
+  siteStateQuery,
+} from './badger-brain';
+
+describe('data/badger-brain backend getSiteState', () => {
+  it('uses the http client to query badger brain', done => {
+    const postFn = (url, query) => {
+      expect(url).to.equal(badgerBrainURL);
+      expect(query).to.equal(siteStateQuery);
+      done();
+    };
+    getSiteState(postFn);
```

Comment:
You can omit the done callback and just return 'getSiteState(postFn)' - preference?